### PR TITLE
glooctl install gateway --dashboard

### PR DESCRIPTION
**Pull request by mitchdraft**
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/1218

### CODE REVIEW

File: pkg/version/enterprise.go [hunk 1]
```diff
@@ -1,14 +1,11 @@
 package version
 
 const (
-	EnterpriseTag = "0.18.29"
+	// When updating the EnterpriseTag value, also update GlooTagInEnterprise
+	EnterpriseTag       = "0.18.31"
```
Comment (GrahamGoudeau) [inline pkg/version/enterprise.go]: we should coordinate releasing this with @rickducott, there's a bunch of observability changes in this that he wanted to test/message out about

Comment (mitchdraft) [reply pkg/version/enterprise.go]: oh I see, thanks for letting me know. Moving to a "WIP" until we're ready to make the bump
This change requires a version of GlooE >= v.0.18.31 so will block until then

**Comment by mitchdraft** [comment]
This change requires a version of GlooE >= v.0.18.31

File: pkg/version/enterprise.go (hunk 1 above)
Comment (rickducott) [reply pkg/version/enterprise.go]: Let's get this in

**Pull request merged**